wasm-engines/docker/ssvm.Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM ewasm/llvm-10:1.0
+
+RUN apt update -y && apt install -y libboost-all-dev && git clone https://github.com/ewasm-benchmarking/SSVM.git --single-branch --branch bench && \
```

Comment:
@hugo-dc @jwasinger should rebase the `bench` branch on top of `0.6.0` and then we can use `-DSSVM_DISABLE_AOT_RUNTIME=ON` to only build the interpreter and remove the need for llvm.